### DISCUSSION TITLE
Why Two GitHub Respositories?

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#1305] [Q&A] [answered]
Why are there two github places for ungoogled-chromium?

1 - https://github.com/Eloston
2 - https://github.com/ungoogled-software

Which one do we trust and use? Or are this two under same team? Because I noticed this; ``Eloston Member of ungoogled-software".

**Comment by Eloston** [comment]
I created ungoogled-software to manage repositories related to ungoogled-chromium. There's no reason why this repo needs to live under my personal account anymore (especially considering we can transfer repos now), but I just haven't gotten around to doing it & update all necessary permissions.

**Reply by daylightbrightledlight** [reply]
Ah, that makes so much more sense! Now I understand. I was always wondering about it and was also a little confused.

**Comment by networkException** [comment, marked as answer]
This repo is now part of ungoogled-software
